### DISCUSSION TITLE
Alternative text on Images

### DISCUSSION THREAD

**Discussion by Jilic-matt** [dolanmiu/docx#1321] [Q&A]
Hi,

Is there a way to add alternative text (descriptions) to an ImageRun? I have been wading through issues/discussions but I cant seem to find anything. This may be a feature request. 

Cheers

**Comment by dolanmiu** [comment]
I believe it is a feature request yes

If possible, could you raise a PR for this feature?

**Reply by Jilic-matt** [reply]
I have had a quick look into it, although im not particularly comfortable in a typescript environment. I think it would be just a matter of adding to the DocProperties constructor when used in Inline and Anchored drawings and back fill to ImageRun

**Reply by Jilic-matt** [reply]
Found here `wp:inline/wp:docPr/_attr`
